### DISCUSSION TITLE
Date Format

### DISCUSSION THREAD

**Discussion by cmrice-arch** [arch-kiosk/arch-kiosk-office#1125] [users: ideas for new features]
Have there been discussions about adding a way to specify, choose, restrict the date format when entering a context / locus? Does the system default to month / day / year vs. day / month / year)? We have this issue periodically with our mix of North Americans and Europeans. It's only confusing a few days a month, but it might be worth thinking about a way to clarify?

**Comment by cmrice-arch** [comment]
Not sure if I need to tag people... @Tfranconi @lbestock

**Comment by lbestock** [comment]
There HAVE been discussions, going way back to the beginning. Lutz can weigh in on what the limitations are from the software perspective (Filemaker will not accept dates in multiple formats, if I remember correctly, so one has to pick one and stick with it). But this is of course always a problem, and we implemented a solution that mirrors the way I have always dealt with it in the field. 

In configuration you have the option to "show additional Latin date" (it's in "visibility"). When selected, that interprets the date put into Filemaker in the US system and makes an extra line that gives the date in the format I have always used on paper. We write the date in the order day/month/year, but we always use a Roman numeral for the month. That way even if someone writes it in the wrong order there is no confusion. So today is 5 VI 21. If that part of configuration is turned on then you get the input field showing the date and time as stamped according to the US system, so 6/5/2021 12:45:34 PM, and below that you get an additional little line saying 5 VI 21. Curious to hear if that would solve your issues. (There is not usually a problem for someone entering the date to screw it up and put the wrong one because usually that's done by hitting the clock button, and the iPad assigns the date for you.)

**Comment by urapadmin** [comment]
For the recording system:
I have never really checked it but my expectation is that the date format one uses for entering depends on the language settings of the local device. So in the US you use the m/d/y while without any changes to software or configuration my ipad in Germany would use d.m.y. That should be true both for entering and reading dates. The latin representation Laurel mentioned is from my perspective a good way to check whether the entered date is in fact what you wanted. 

As for Kiosk: 
There are only a few places where we show or enter dates. I have tried to allow for even entering the latin dates (filemaker won't allow that) e.g. in the file repository / edit dialog. But there is absolutely no coherence in kiosk in this regard. I would expect kiosk to stick to the language (rather region) settings of the Browser (which usually defaults to the OS) but again, I have never tested this or payed much attention to the whole date shebang since it would be part of the timezone revision of a version 2 of both the recording and kiosk. Then I will do it right :o) And as for right: Then we should get together and think about what that would mean.
